### ISSUE TITLE
ACF pro & Yoast conflict

### ISSUE THREAD

**Issue by PascalMarechal** [AdvancedCustomFields/acf#518]
Hi,

I have latest ACF pro (5.9.7) and Yoast (16.5)

When I try to update my ordered posts I get this issue :

![Issue](https://user-images.githubusercontent.com/5609628/123262396-1f7a0380-d4f8-11eb-9f6b-a7b1aaf3f41d.png)

If you want to get a view of my backoffice :
![Screenshot 2021-06-24 at 14-26-43 Edit Page ‹ The Coaches' Voice — WordPress](https://user-images.githubusercontent.com/5609628/123262534-459fa380-d4f8-11eb-9562-4fa4bf22e76c.png)

The bug happens when I try to change posts order.

If I downgrade Yoast to 16.3 there is no issue at all.

Thanks for your help.

**Comment by PascalMarechal** [comment]
OK I have more info about this but no fix so far.
These lines are the ones which throw an error in acf-input.js. Around line 7677.

`// sortable
          $ul.sortable({
            stop: function (e) {
              // loop
              $ul.find('.select2-selection__choice').each(function () {
                // vars
                console.log($(this))
                console.log($(this).data('data')) // <= THIS is not defined and the sorting is not working...
                var $option = $($(this).data('data').element)
  
                // detach and re-append to end
                $option.detach().appendTo($select)
              })
  
              // trigger change on input (JS error if trigger on select)
              $select.trigger('change')
            },
          })`

**Comment by JiveDig** [comment]
Hitting this as well on one site, but I couldn't recreate on another. Are you able to see this same error on a base install with only ACF and Yoast?

**Comment by lgladdy** [comment]
Hey @JiveDig and @PascalMarechal,

We've fixed this in our next release, 5.10, due for release next week.

Thanks for the report!

**Comment by JiveDig** [comment]
Great! Happy to test too. We now have 2 live sites that hit this issue consistently, though we haven't hit it elsewhere using the same tools.

**Comment by lgladdy** [comment]
@JiveDig That's great! We're planning a release candidate build later today or tomorrow that we're going to post here on GitHub to give our more technical users a preview! I'll make sure we mention you when we publish it!

**Comment by lgladdy** [comment]
@JiveDig Hey! We've published ACF Pro 5.10-RC1 which includes this fix. You can download it from the My Account section on [advancedcustomfields.com](https://www.advancedcustomfields.com/my-account/view-licenses/).

If you don't have an ACF Pro licence, drop us an email to the support address, and I'll send you a ACF Free 5.10-RC build that way.

**Comment by JiveDig** [comment]
Hit some other stuff mentioned in another Issue, but this one is working for me with 5.10-RC1. Thanks!
